.github/workflows/test_branches.yml

```diff
@@ -71,29 +71,29 @@ jobs:
       fail-fast: false
```

Comment:
There is now _NO_ Python 3.9 testing in this file, and I think that's an issue. Maybe win/pip should have it / cover it?